Sources/BraintreeApplePay/BTApplePayCardNonce.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+/// Contains information about a tokenized Apple Pay card.
+@objcMembers public class BTApplePayCardNonce: BTPaymentMethodNonce {
+
+    /// The BIN data for the card number associated with this nonce.
+    let binData: BTBinData?
```

Comment:
Should this be public?

Reply:
Good catch, yes it should

Reply:
Also, looks like BTBinData was [non-optional](https://github.com/braintree/braintree_ios/blob/dd0cbeae0b48606885ed3684b7463ad799d8ee13/Sources/BraintreeApplePay/Public/BraintreeApplePay/BTApplePayCardNonce.h#L1) before

Reply:
Done: https://github.com/braintree/braintree_ios/pull/956/commits/c71b1a7f3bdffadddab2ec3a177213c102b808c5

Reply:
why does it need to be public? for testing?

Reply:
Jax, didn't we decide to make this optional because it objective-c, this property could be nil?

Reply:
That's a great question! It needs to be public so that merchants can use it in their integration. In the link Sammy posted [above](https://github.com/braintree/braintree_ios/pull/956#discussion_r1109889198) in Obj-C we declare this files as "assume nonnull". This means that in Obj-C we were assuming we'd never get a null object back here. This is a different pattern from the other nonces in Obj-C [here is PayPal as an example](https://github.com/braintree/braintree_ios/blob/5.x/Sources/BraintreePayPal/Public/BraintreePayPal/BTPayPalAccountNonce.h) where each of the properties are `nullable`.

Reply:
Thanks Jax!